Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -681,7 +681,7 @@ internal async Task<bool> IsSafeToProceed( EventCallback<CancellableRowChange<TI
 
         #region Filtering
 
-        protected void HandleSortColumn( DataGridColumn<TItem> column )
+        protected void HandleSortColumn( DataGridColumn<TItem> column, bool initializing )

```

Comment:
Maybe it could be named `changeDirection` or `switchDirection` which is the real purpose of the flag. To be able to call the method and decide if we should change or not sort direction. Which we do not want to, when initializing. Not a big deal in my opinion.

Reply:
Done